Formula/morse.rb

```diff
@@ -24,12 +24,20 @@ class Morse < Formula
   end
 
   depends_on "pkg-config" => :build
-  depends_on "pulseaudio"
+
+  on_macos do
+    depends_on "libx11"
+  end
+
+  on_linux do
+    depends_on "pulseaudio"
+  end
 
   def install
-    system "make", "all"
-    bin.install %w[morse QSO]
-    man1.install %w[morse.1 QSO.1]
+    inreplace "morse.d/Makefile", "/usr/X11R6/", "#{Formula["libx11"].opt_prefix}/"
+    system "make", "all", "DEVICE=#{OS.mac? ? "X11" : "PA"}"
```

Comment:
X11 is what upstream recommends on macOS. May need to check test as it stalled on my local machine due to having XQuartz installed. Not sure if anything is pre-installed on CI runners.

https://gitlab.com/esr/morse-classic/-/blob/master/Makefile#L14-15
```
# Running on Mac OS X:
# (1) Use X11.  The user must, as with any X11 client, set the DISPLAY 
```